VAMobile/src/components/Menu/MenuView.tsx

```diff
@@ -29,11 +31,9 @@ export type MenuItemActionsType = {
   /** adds a divider after the action*/
   addDivider: boolean
   /** name of the icon to show */
-  iconName?: keyof typeof VA_ICON_MAP
+  iconProps?: IconProps
   /** method to  */
```

Comment:
This existing comment must have been truncated previously

Reply:
updated